cobra/test/conftest.py

```diff
@@ -74,6 +74,14 @@ def salmonella(medium_model):
     return medium_model.copy()
 
 
+@pytest.fixture(scope="session")
```

Comment:
1. Please move this fixture into the `cobra/test/test_io/test_json.py` module.
2. I outlined how to load the schema with importlib resources. Did you try that?

Reply:
I did try using 'importlib_resources' also. The tests were passing but the tox commands were failing with the error: ImportError: No module named importlib_resources